packages/back-end/src/routers/organizations/organizations.controller.ts

```diff
@@ -1747,3 +1747,28 @@ export async function putLicenseKey(
     status: 200,
   });
 }
+
+export function putDefaultRole(
+  req: AuthRequest<{ defaultRole: MemberRole }>,
+  res: Response
+) {
+  const { org } = getOrgFromReq(req);
+  const { defaultRole } = req.body;
+
+  req.checkPermissions("manageTeam");
```

Comment:
Add a check for commercial features here

Reply:
@jdorn Just pushed up this [commit](https://github.com/growthbook/growthbook/pull/1737/commits/8d3aedce723a32872b4b32641565bba25f9c3b44) - is this what you mean? I didn't see a pattern in the app for checking for commercial features via `checkPermissions`

Reply:
I recently had to implement a commercial features check for our features REST API endpoints; here is a link to the code. My guess is that we'd want to check for the `'advanced-permissions'` feature. Let me know if this doesn't help!
https://github.com/growthbook/growthbook/blob/9dafde41475c6cd284df345cbb570c26214360a5/packages/back-end/src/api/features/postFeature.ts#L51-L52

Reply:
@bttf Ah - thank you for the suggestion! Very appreciated! Just updated to utilize this pattern, and checks against the "sso" feature as that's the mechanism that currently gates SCIM.